hr_payroll_analysis/wizard/payroll_analysis.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding:utf-8 -*-
+import pytz
+from datetime import datetime, date
+
+from openerp import models, fields, api, _
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+def get_current_year(context):
+    if not context:
+        context = {}
+    tz = context.get('tz', False)
+    tz = tz and pytz.timezone(tz) or pytz.utc
+    return datetime.now(tz).year
+
+
+class PayrollAnalysis(models.TransientModel):
+    """
+    Wizard to generate payroll analysis report
+    """
+    _name = 'hr.payroll.analysis'
+    _description = 'Payroll Analysis Wizard'
+
+    type = fields.Selection(
```

Comment:
please use another name for this parameter. type is a reserved word in python. This could cause subtile problems.